server/metrics.go

```diff
@@ -73,5 +75,14 @@ func NewServerMetrics(cfg Config) *Metrics {
 			Name:      "inflight_requests",
 			Help:      "Current number of inflight requests.",
 		}, []string{"method", "route"}),
+		SlowRequestServerThroughput: reg.NewHistogramVec(prometheus.HistogramOpts{
+			Namespace:                       cfg.MetricsNamespace,
+			Name:                            "slow_request_server_throughput_" + cfg.ThroughputConfig.Unit,
```

Comment:
I am not sure about the `_server_` name. Doesn't everything measured here refer to the server? (same goes for for the field name)